src/main/java/org/jitsi/jigasi/transcription/WhisperTsAudioSilenceCaptureDevice.java

```diff
@@ -0,0 +1,224 @@
+package org.jitsi.jigasi.transcription;
+
+import java.io.IOException;
+import java.util.Arrays;
+import javax.media.Buffer;
+import javax.media.Format;
+import javax.media.control.FormatControl;
+import javax.media.format.AudioFormat;
+import javax.media.protocol.BufferTransferHandler;
+import org.jitsi.impl.neomedia.codec.AbstractCodec2;
+import org.jitsi.impl.neomedia.jmfext.media.protocol.AbstractPushBufferCaptureDevice;
+import org.jitsi.impl.neomedia.jmfext.media.protocol.AbstractPushBufferStream;
+import org.jitsi.impl.neomedia.jmfext.media.renderer.audio.AbstractAudioRenderer;
+import org.jitsi.utils.logging.Logger;
+
+class WhisperTsAudioSilenceCaptureDevice extends AbstractPushBufferCaptureDevice {
+    private static final long CLOCK_TICK_INTERVAL = 20L;
+    private static final Format[] SUPPORTED_FORMATS;
+    private final boolean clockOnly;
+
+    private final static Logger logger
+            = Logger.getLogger(WhisperTsAudioSilenceCaptureDevice.class);
+
+    WhisperTsAudioSilenceCaptureDevice(boolean clockOnly) {
+        this.clockOnly = clockOnly;
```

Comment:
Please run the check style, this for sure should fail ({ should be on new line). If it doesn't fail we need to fix the check style config.